pytest_mccabe.py

```diff
@@ -24,6 +24,11 @@ def pytest_addoption(parser):
              "deactivate checking). example: *.py 10")
 
 
+def pytest_configure(config):
+    if config.option.mccabe:
+        config.addinivalue_line('markers', 'mccabe: Tests which run mccabe')
```

Comment:
I think it should always add the `ini` value, otherwise I suspect users passing only `--strict` and not `--mccabe` in the command line will see errors about the marker being used in the tests but not being declared.